docs/changes/1.1.2.rts

```diff
@@ -0,0 +1,18 @@
+Changelog for QTT 1.1.2
+==========================
+
+This release changes the folder structure and starts seperating the unittests from the source code.
```

Comment:
Actually we're finished seperating unit tests from the source code. pytest is also not looking for tests in the source directory anymore (see pytest.ini). Further Coverage is only counting code coverage in directory src/qtt, not the src/tests directory (see .travis.yml)